CHANGES.rst

```diff
@@ -242,6 +242,8 @@ astropy.time
   existing method for ``TimeDelta``) so that one can get values in a given
   ``format`` and possible ``subfmt`` (e.g., ``to_value('mjd', 'str')``. [#9361]
 
+- ``Time`` will correctly display __repr__() values. [#9525]
```

Comment:
You might want to add a bit more detail in this note to say that it's specifically about long lists, since "correctly" is a bit ambiguous.

Reply:
This should be more like:
```
  - ``Time`` now displays a maximum of 50 array values in the representation before triggering
    summarization. [#9525]
```